### PR TITLE
Several small fixes to documentations

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: doc/rst/source/GMT_Docs.rst [hunk 1]
```diff
@@ -707,7 +706,7 @@ modify the appearance of plots or affect the manipulation of data. When
 a program is run, it initializes all parameters to the
 GMT\ defaults [9]_, then tries to open the file ``gmt.conf`` in the current
 directory [10]_. If not found, it will look for that file in a
-sub-directory ``/.gmt`` of your home directory, and finally in your home directory
+sub-directory ``~/.gmt`` of your home directory, and finally in your home directory
```
Comment (joa-quim) [inline doc/rst/source/GMT_Docs.rst]: The ~ is unix specific. I think this one should stay as is.

File: doc/rst/source/GMT_Docs_classic.rst [hunk 2]
```diff
@@ -707,7 +706,7 @@ modify the appearance of plots or affect the manipulation of data. When
 a program is run, it initializes all parameters to the
 GMT\ defaults [9]_, then tries to open the file ``gmt.conf`` in the current
 directory [10]_. If not found, it will look for that file in a
-sub-directory ``/.gmt`` of your home directory, and finally in your home directory
+sub-directory ``~/.gmt`` of your home directory, and finally in your home directory
```
Comment (joa-quim) [inline doc/rst/source/GMT_Docs_classic.rst]: Same here

File: doc/rst/source/GMT_Docs.rst (hunk 1 above)
Comment (joa-quim) [inline doc/rst/source/GMT_Docs.rst]: The ~ is unix specific. I think this one should stay as is.

File: doc/rst/source/GMT_Docs_classic.rst (hunk 2 above)
Comment (joa-quim) [inline doc/rst/source/GMT_Docs_classic.rst]: Same here

File: doc/rst/source/GMT_Docs.rst (hunk 1 above)
Comment (seisman) [reply doc/rst/source/GMT_Docs.rst]: Can we change `/.gmt` to `.gmt`?

Comment (joa-quim) [reply doc/rst/source/GMT_Docs.rst]: Hmm, why? Doesn't a slash give a more clear indication that .gmt is a directory?

Comment (seisman) [reply doc/rst/source/GMT_Docs.rst]: OK. I've changed it back to `/.gmt`.

**Pull request merged**